### PR TITLE
Add Grid Intensity CLI Docs

### PR DESCRIPTION
**Pull request by fershad**
Add an initial set of documentation introducting and explaing the Grid Intensity CLI.

This PR will include:

- [ ] Overview
- [ ] Installation
- [ ] Explainer: Information about providers
- [ ] Tutorial: Getting grid intensity information for a region
- [ ] Tutorial: Exporting grid intensity data

### CODE REVIEW

**Comment by fershad** [comment]
[Go CLI - Overview](https://trello.com/c/i75ub2ph/45-go-cli-overview)

**Comment by mrchrisadams** [comment]
thanks for this Fish! Can you @ message me when you'd like me to review this for merging in?

And is it ok to merge in myself if I'm happy?

**Comment by fershad** [comment]
@mrchrisadams this PR is ready for review. The first set of docs for the Grid Intensity CLI can be previewed at the below branch preview URL:

https://add-grid-intensity-cli-docs.developer-docs-64x.pages.dev/

No worries if you want to merge changes yourself, or if there's feedback you can leave it here.

File: src/docs/grid-intensity-cli/tutorials/getting-grid-intensity.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Overview
+
+Knowing the grid intensity for the regions in which your code runs allows you to make better informed decisions about where and/or when to run it.
+
+In this tutorial, you will use the Grid Intensity CLI to:
+
+1. Get historical intensity data for a region.
+1. Change data providers, and
+1. Get near 0real-time grid intensity data for a region.
```
Comment (mrchrisadams) [inline src/docs/grid-intensity-cli/tutorials/getting-grid-intensity.md:21]: tiny typo here

**Comment by mrchrisadams** [comment]
hi @fershad  I'm happy with this - there's just one typo which I've highlighted, can you please merge this in after making that one change?

File: src/docs/grid-intensity-cli/tutorials/getting-grid-intensity.md (hunk 1 above)
Comment (fershad) [reply src/docs/grid-intensity-cli/tutorials/getting-grid-intensity.md]: Fixed in https://github.com/thegreenwebfoundation/developer-docs/pull/6/commits/b7567fc2fd5a05a983acd5df116ca058f5f77393

**Pull request merged**